src/test/rustdoc/issue-76501.rs

```diff
@@ -0,0 +1,18 @@
+#![feature(const_fn)]
+
+// @has 'issue_76501/fn.bloop.html' '//pre' 'pub const fn bloop() -> i32'
+/// A useless function that always returns 1.
+pub const fn bloop() -> i32 {
+    1
+}
+
+/// A struct.
+pub struct Struct {}
+
+impl Struct {
+    // @has 'issue_76501/struct.Struct.html' '//*[@class="method"]' 'pub const fn bloop() -> i32'
+    /// A useless function that always returns 1.
+    pub const fn bloop() -> i32 {
```

Comment:
nit: can you name this something other than `bloop` on the off chance it ends up in the same file as the free function? The line you put the annotation on doesn't actually affect the test, so this avoids false negatives: https://github.com/rust-lang/rust/pull/73101#discussion_r453274499

Reply:
Sure, will do.